internal/services/network/route_table_data_source.go

```diff
@@ -106,6 +111,10 @@ func dataSourceRouteTableRead(d *pluginsdk.ResourceData, meta interface{}) error
 		if err := d.Set("subnets", flattenRouteTableDataSourceSubnets(props.Subnets)); err != nil {
 			return err
 		}
+
+		if err := d.Set("disable_bgp_route_propagation", props.DisableBgpRoutePropagation); err != nil {
```

Comment:
```suggestion
		if err := d.Set("bgp_route_propagation_enabled", !props.DisableBgpRoutePropagation); err != nil {
```

Reply:
Thanks, let me check if I can apply this.